File: app/src/main/java/android/itacademy/by/dz14/imageloader/ImageLoaderUtil.java
package android.itacademy.by.dz14.imageloader;

import android.itacademy.by.menu.R;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public class ImageLoaderUtil {
    public static void loadImage(ImageView imageView, String url) {

        Picasso.get()
                .load(url)
                .placeholder(R.drawable.progress)
                .error(R.drawable.inset_oval)
                .into(imageView);
    }
}

File: app/src/main/java/android/itacademy/by/dz14/imageloader/ImageLoaderUtil.kt
package android.itacademy.by.dz14.imageloader

import android.itacademy.by.menu.R
import android.widget.ImageView

import com.squareup.picasso.Picasso

object ImageLoaderUtil {
    fun loadImage(imageView: ImageView, url: String) {

        Picasso.get()
                .load(url)
                .placeholder(R.drawable.progress)
                .error(R.drawable.inset_oval)
                .into(imageView)
    }
}